manifests/crc/default-db-included-installation/minikube/kruize-crc-minikube.yaml

```diff
@@ -54,7 +54,7 @@ spec:
             - name: POSTGRES_USER
               value: admin
             - name: POSTGRES_DB
-              value: kruizeDB
+              value: kruizeDB2
```

Comment:
why KruixeDB2 ?

Reply:
Forgot to remove after testing. Updated now